studygroups/views/facilitate.py

```diff
@@ -472,3 +472,19 @@ def form_valid(self, form):
             TeamMembership.objects.create(team=invitation.team, user=self.request.user, role=invitation.role)
 
         return super(InvitationConfirm, self).form_valid(form)
+
+
+class StudyGroupFacilitatorFeedback(TemplateView):
```

Comment:
@s-kennedy this view is behind a login, iow, we should check that the user is indeed the facilitator. Also, I don't think we need to use UUIDs for this, since someone can't get to the survey by guessing the id. I'll update